CONTRIBUTING.md

```diff
@@ -2,10 +2,10 @@
 
 ## Setup
 
-Run `npm install` to install the dependencies.
+Run `bun install` to install the dependencies.
 
-Run the tests with `npm run test`.
+Run the tests with `bun run test`.
 
-Run the linter with `npm run lint`.
+Run the linter with `bun run lint`.
```

Comment:
There's not `lint` target, but `quality`.

Regarding this, `bun run quality` gives me 147 errors. Is it same for you? Can you create a commit to format it?